src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -0,0 +1,87 @@
+|          f32           | The base ABI uses 32-bit FPRs for parameter passing. (lp64f ilp32f)|
+|          sf            | The base ABI uses no FPR for parameter passing. (lp64s ilp32s)     |
+
+|   `<abiext-suffix>`    |                       `ABI extension type`                         |
+|------------------------|--------------------------------------------------------------------|
+|    (empty string)      |                               base                                 |
+
+|`ABI type(Base ABI/ABI extension)`| `C library` | `kernel` |          `target tuple`          |
+|----------------------------------|-------------|----------|----------------------------------|
+|           lp64d/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf64 |
+|           lp64f/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf32 |
+|           lp64s/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnusf  |
+|           ilp32d/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf64 |
+|           ilp32f/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf32 |
+|           ilp32s/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnusf  |
+|           lp64d/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf64|
+|           lp64f/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf32|
+|           lp64s/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslsf |
+|           ilp32d/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf64|
+|           ilp32f/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf32|
+|           ilp32s/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslsf |
+
+## Target maintainers
+
+- zhaixiaojuan, `zhaixiaojuan@loongson.cn`, https://github.com/zhaixiaojuan
+
+## Requirements
+
+This target is cross-compiled.
+A platform-provided C compiler toolchain is required, and obtaining address of the cross toolchain [GNU LoongArch toolchain](https://github.com/loongson/build-tools/releases).
+
+## Building the target
+
+The target can be built by enabling it for a `rustc` build.
+
+```toml
+[build]
+target = ["loongarch64-unknown-linux-gnuf64"]
+```
+
+Make sure `loongarch64-unknown-linux-gnu-gcc` is included in `$PATH`. Alternatively, you can use GNU LoongArch Toolchain by adding the following to `config.toml`:
+
+```toml
+[target.loongarch64-unknown-linux-gnuf64]
+cc = "loongarch64-unknown-linux-gnu-gcc"
+cxx = "loongarch64-unknown-linux-gnu-g++"
+ar = "loongarch64-unknown-linux-gnu-ar"
+ranlib = "loongarch64-unknown-linux-gnu-ranlib"
+linker = "loongarch64-unknown-linux-gnu-gcc"
+```
+
+## Cross-compilation
+This target can be cross-compiled as `x86_64`, and cross-compilation on other hosts is not currently supported.
```

Comment:
```suggestion
This target can be cross-compiled on a `x86_64-unknown-linux-gnu` host.  Cross-compilation on other hosts may work but is not tested.
```

Reply:
"This target can be cross-compiled. Cross-compiling from `x86_64-unknown-linux-gnu` is known to work; any other host will likely work too."